src/collections/news/2023-05-17-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -0,0 +1,136 @@
+of the single Bliss symbols were used as the training set.  These are shown
+in an array of symbols in Figure 1 below.
+
+<figure style="text-align: center;">
+<img src="/news/images/StyleGAN2-ADA-reals.png" alt="array of real single
+character Bliss symbols">
+<figcaption>Figure 1. Array of Bliss symbols used to train StyleGAN2-ADA</figcaption>
+</figure>
+
+Guided by the article [How to Train StyleGAN2-ADA with Custom Dataset](https://towardsdatascience.com/how-to-train-stylegan2-ada-with-custom-dataset-dc268ff70544),
+a StylGAN2-ADA network was trained over several runs for total training
+time of 62 hours.  The following figure shows an array of the final synthesized
+images generated by the GAN.  Note that these images do not correspond to real
+Bliss symbols but are produced by StyleGAN2-ADA.  The GAN extracts features or
+patterns found in the training data and attempts to mimic Bliss symbols based
+on the learned features.
+
+<figure style="text-align: center;">
+<img src="/news/images/StyleGAN2-ADA-fakes.png" alt="array of fake Bliss
+symbol images as generated by StyleGAN2-ADA">
+<figcaption>Figure 2. Array of images generated by StyleGAN2-ADA</figcaption>
+</figure>
+
+As can be seen, the generated images have a superficial resemblance to real
+Bliss symbols and it appears that the GAN has detected and learned some features
+or patterns present in the single Bliss symbols. At the present time it is
+unclear what patterns the GAN has learned exactly.  This is an issue for
+potential future explorations of GAN machine learning models.
+
+### StyleGAN3
+
+- Cindy to help here
+
+## CLIP model
+
+CLIP (Contrastive Language–Image Pre-training) comprises a machine learning
+technique in which images, in addition to textual information describing those
+images are used in the training process. This approach has been recieving
+increased attention in the field of computer vision, where previous advances
+have provided high levels of accuracy in tasks such as the recognition and
+classification of objects in photographs and videos. However, it has been argued
+that state-of-the-art approaches in computer vision have various limitations,
+including for example that the systems are often able to make predictions and
+identification over a small number of categories, preventing the ability to
+identify more general and abstract features, not to mentioned the laborious work
+required in the preparation of the data when the systems are required to
+recognize new objects not previously included in the datasets.
+
+The BBB Project has started experimenting with CLIP models in order to explore
+how the use of multimodal (images and textual) data training strategies can be
+applied for a better understanding of the internal composition of Bliss symbols
+and combined words, and eventually provide avenues for the generation of new
+symbols.
+
+For this exploration, we have created an initial annotated dataset using the
```

Comment:
@agamba, it feels the section for the CLIP model ends at a unexpected spot where you mentioned the dataset is ready. What about next steps: the model training with this dataset, findings/results from this experiment etc. If you are currently working on these tasks, it would be helpful to mention that to conclude the section.